source/splitWhenever.js

```diff
@@ -6,10 +6,9 @@ import _curryN from './internal/_curryN';
  * @memberOf R
  * @since v0.26.1
  * @category List
- * @sig (a -> Boolean) -> [a] -> [[a]]
- * @param {Number} n
- * @param {Array} list
- * @param {Boolean} incDelimiter
+ * @sig (a -> Boolean) -> [a] -> []
```

Comment:
Thanks for the update to the params, however I think the @sig was correct initially. Can you change it back?

Reply:
Sure.

Reply:
Done